dashboard/modules/runtime_env/runtime_env_agent.py

```diff
@@ -412,22 +416,23 @@ async def _create_runtime_env_with_retry(
             self._env_locks[serialized_env] = asyncio.Lock()
 
         async with self._env_locks[serialized_env]:
+            runtime_env_context = None
+            runtime_env_reply = None
             if serialized_env in self._env_cache:
-                serialized_context = self._env_cache[serialized_env]
                 result = self._env_cache[serialized_env]
                 if result.success:
-                    context = result.result
+                    runtime_env_context = result.context
                     self._logger.info(
                         "Runtime env already created "
-                        f"successfully. Env: {serialized_env}, "
-                        f"context: {context}"
+                        f"successfully. Env: {serialized_env}."
                     )
-                    return runtime_env_agent_pb2.GetOrCreateRuntimeEnvReply(
-                        status=agent_manager_pb2.AGENT_RPC_STATUS_OK,
-                        serialized_runtime_env_context=context,
+                    runtime_env_reply = (
+                        runtime_env_agent_pb2.GetOrCreateRuntimeEnvReply(
+                            status=agent_manager_pb2.AGENT_RPC_STATUS_OK,
+                        )
```

Comment:
Because I change the type of context from "str" to "RuntimeEnvContext" and we should rewrite the context in `pre_worker_startup`. So we can't put it into the reply here.